presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDatabaseMetaData.java

```diff
@@ -94,6 +108,67 @@ public void testPassEscapeInMetaDataQuery()
         assertContains(query, "_t' ESCAPE '", "Metadata query does not contain ESCAPE");
     }
 
+    @Test
+    public void testGetTypeInfo()
+            throws Exception
+    {
+        ImmutableSet<Integer> typesToCheck = ImmutableSet.of(Types.BIGINT, Types.BOOLEAN, Types.INTEGER, Types.DECIMAL, Types.VARCHAR, Types.TIMESTAMP, Types.DOUBLE);
+        DatabaseMetaData metaData = connection.getMetaData();
+        ResultSet typeInfo = metaData.getTypeInfo();
+        while (typeInfo.next()) {
+            int jdbcType = typeInfo.getInt("DATA_TYPE");
+            if (typesToCheck.contains(jdbcType)) {
+                switch (jdbcType) {
+                    case Types.BIGINT:
+                        assertColumnSpec(typeInfo, Types.BIGINT, 19L, 10L, BIGINT.getDisplayName());
```

Comment:
Let's hard-code all the expected type name strings. It's hard to tell that the tests are doing the right thing.